Mathlib/Topology/Homotopy/HomotopyGroup.lean

```diff
@@ -447,11 +447,14 @@ def HomotopyGroup.pi0EquivZerothHomotopy : π_ 0 X x ≃ ZerothHomotopy X :=
 /-- The 1-dimensional generalized loops based at `x` are in bijection with loops at `x`. -/
 def genLoopEquivOfUnique (N) [Unique N] : Ω^ N X x ≃ Ω X x where
   toFun p :=
-    Path.mk ⟨fun t => p fun _ => t, by continuity⟩
+    Path.mk ⟨fun t => p fun _ => t, by
```

Comment:
My impression is that this version is significantly faster. That said, it would be great to reduce this to one line... perhaps mk_apply should be tagged with fun_prop?